packages/react-sdk/src/useEnsAddress.ts

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react'
+import { useSyncAgent } from './useSyncAgent'
+import { useObservable } from './useObservable'
+import { useAction } from './internals/useAction'
+
+export const useEnsAddress = (spaceId: string, userId: string) => {
+    const sync = useSyncAgent()
+    const member = useMemo(
+        () => sync.spaces.getSpace(spaceId).members.getMember(userId),
+        [sync, spaceId, userId],
+    )
+    const { data, ...rest } = useObservable(member?.observables.ensAddress)
+    return {
+        ...data,
+        ...rest,
+    }
+}
+
+export const useSetEnsAddress = (spaceId: string) => {
+    const sync = useSyncAgent()
+    const member = useMemo(
+        () => sync.spaces.getSpace(spaceId).members.getMember(sync.userId),
+        [sync, spaceId],
+    )
+    const { action: setEnsAddress, ...rest } = useAction(member, 'setEnsAddress')
+    return { setEnsAddress, ...rest }
```

Comment:
1) I think we can probably figure out way to have useAction return an object with a key named how we want so we don't have to deconstruct and reconstruct this object, but otherwise i think we need useMemo again